### PR TITLE
[WIP] update interop examples

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

File: GLMakie/experiments/cuda_interop.jl [hunk 1]
```diff
@@ -1,11 +1,68 @@
-using CUDA, GLMakie
+using CUDA, GLMakie, NVTX
+using GLMakie.GLAbstraction
+# from https://discourse.julialang.org/t/cuarray-glmakie/52461/11?u=maleadt
 
+function plot(; T=Float32, N=1024, resolution=(800, 600))
```
Comment (asinghvi17) [inline GLMakie/experiments/cuda_interop.jl:5]:
```suggestion
function cu_plot(; T=Float32, N=1024, resolution=(800, 600))

File: GLMakie/experiments/cuda_interop.jl [hunk 2]
```diff
@@ -1,11 +1,68 @@
...
+    end
+
+    ## clean-up
+
+    CUDA.cuGraphicsUnregisterResource(resource)
+
+    return
+end
+
+plot()
```
Comment (asinghvi17) [inline GLMakie/experiments/cuda_interop.jl:68]:
```suggestion
cu_plot()

**Comment by SimonDanisch** [comment]
Would be nice to improve the example further, but will merge this for now to not have too many dangling PRs! It's definitely an improvement over the old "example" :)

File: GLMakie/experiments/cuda_interop.jl [hunk 3]
```diff
@@ -1,11 +1,68 @@
...
+
+            CUDA.cuGraphicsUnmapResources(1, [resource], stream())
+        end
+
+        # generate and render plot
+        NVTX.@range "Makie" begin
+            scatter!(ax, buffer)
+            # force everything to render (for benchmarking purposes)
+            GLMakie.render_frame(screen; resize_buffers=false)
+            GLMakie.glFinish()
```
Comment (asinghvi17) [inline GLMakie/experiments/cuda_interop.jl:57]: How would you update values in the buffer (say for an expensive computation which has to run interactively)?

**Pull request merged**